pkgs/tools/networking/lprobe/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  libpcap,
+}:
+
+buildGoModule rec {
+  pname = "lprobe";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "fivexl";
+    repo = "lprobe";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-q7lH0aLgQNM4jrrrq2ua+pt4/VknxlzKzDH5J4MwjfA=";
+  };
+
+  vendorHash = "sha256-B3lcE33Ny+XE7nK/QlVcV8yYgzYWNBfoecuL+AcavSk=";
+
+  buildInputs = [
+    libpcap
+  ];
+
+  meta = with lib; {
+    description = "A command-line tool to perform Local Health Check Probes inside Container Images (ECS, Docker)";
+    homepage = "https://github.com/fivexl/lprobe";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ cageyv ];
+    mainProgram = "lprobe";
+  };
+}
```

Comment:
This file is a duplicate, please remove it - `pkgs/by-name` should be preferred when creating new packages